app/views/ops/_logs_selected.html.haml

```diff
@@ -4,7 +4,12 @@
 - last_collection_time = last_time.blank? ? _("Never") : format_timezone(last_time.to_time, Time.zone, "gtl")
 
 #selected_div
-  - if @sb[:selected_server_id] != my_server.id && @selected_server.started? == false
+  - serverStarted = true
+  - begin
+    - serverStarted = @selected_server.started?
+  - rescue => e
```

Comment:
What is the error?  I'd rather we didn't band aid over an error if there is a more underlying problem.